src/core/setup-application.ts

```diff
@@ -409,7 +297,7 @@ export default class CreateApplication extends BaseProject {
                 if (url) {
                     AppLogger.debug(`Deleting repository for..., ${url}`);
                     const command = `curl -X DELETE -u "${userName}:${token}" ${url}`;
-                    const result = execSync(command, { stdio: 'pipe' });
+                    const result = executeCommandWithRetry(command, { stdio: 'pipe' })
                     AppLogger.debug(`Repository deleted successfully:, ${result.toString()}`);
```

Comment:
what is the value of result? also, without checking it how is it considered to be deleted successfully...

Reply:
I have added the try catch block over it. Now we are checking if there is any error during deleting the repositories.